### ISSUE TITLE
Cannot start Voyager because of name of user directory

### ISSUE THREAD

**Issue by fbanning** [queryverse/DataVoyager.jl#45]
``` julia
julia> using DataVoyager

julia> Voyager()

ERROR: Path contained unexpected character
Stacktrace:
 [1] error(::String) at .\error.jl:33
 [2] parse_url(::String) at C:\Users\Frederik Banning\.julia\packages\URIParser\IuWoB\src\parser.jl:278
 [3] URI at C:\Users\Frederik Banning\.julia\packages\URIParser\IuWoB\src\parser.jl:312 [inlined]
 [4] Voyager() at C:\Users\Frederik Banning\.julia\packages\DataVoyager\71yWt\src\DataVoyager.jl:23
 [5] top-level scope at REPL[5]:1
```

I suppose the problem might have to do with the space in my user directory path. Is this assumption correct? What can we do to avoid such an error?

**Comment by MikaelFiil** [comment]
I seem to have the same problem, did you solve it?

**Comment by fbanning** [comment]
No, I didn't solve it. I simply stopped trying to use DataVoyager and resorted to other means of data exploration. Sorry.

**Comment by ryanbrownnetworking777** [comment]
I have this same issue, is there any resolution on this?

**Comment by fbanning** [comment]
I have not found one and stopped trying to use DataVoyager.jl as a result because I couldn't get it working.

**Comment by ryanbrownnetworking777** [comment]
@fbanning I figured out what was the problem at least for me. Data Voyager for me wasn't properly encoding spaces and the URI library wasn't able to parse the string. I have spaces in my user directory as well as the environment name I was using at the time, so Data Voyager wouldn't work. I've updated part of the DataVoyager.jl code on my local machine to replace spaces with the proper web encodings. I'm going to see if I can't get a change for this done and submit for a PR. Because this really is an issue that should be fixed.
